pubtools/pulplib/_impl/model/repository/base.py

```diff
@@ -303,6 +303,36 @@ class Repository(PulpObject, Deletable):
     .. versionadded:: 2.29.0
     """
 
+    include_in_download_service = pulp_attrib(
```

Comment:
The commit message mentioned the need to set the field, but if you want to be able to do it with this library you'll have to set mutable=True in here.  See `product_versions` field as an example.  Do you want to include that in this change?